### ISSUE TITLE
-l list file types option broken

### ISSUE THREAD

**Issue by jalapenopuzzle** [ianka/xfs_undelete#48]
Tested at commit c01b5fac55966a517f40a4aa3c3593e7f061502f (v13.1-2-g4278cf7).

The "list filetypes" option -l documented in the man page does not work as advertised:

```
[user@host xfs_undelete]# xfs_undelete -l
bad option "-stride": must be -ascii, -command, -decreasing, -dictionary, -increasing, -index, -indices, -integer, -nocase, -real, or -unique
    while executing
"lsort -dictionary -stride 3 -index 0 $::filetypes"
    invoked from within
"if {[dict get $::parameters l]} {
	## Yes. Get file types understood by the file utility
	if {![catch {exec -ignorestderr -- file -l {*}$magicopts 2>/..."
    (file "/root/XFS_Data_Recovery/xfs_undelete_20240111/xfs_undelete/xfs_undelete" line 439)
[user@host xfs_undelete]#
```

Testing with git bisect reveals that this output is produced since commit a867fdfa9172b8132935a2243179fc0d53fab7e9 "added -l option for listing understood file extensions". Perhaps this option never worked properly, or there is something broken on my system.

**Comment by ianka** [comment]
Chances are you are using Tcl-8.5. That one does not yet support `lsort -stride`.

As I try to support Tcl-8.5 lately, mainly for older CentOS versions, I have a look into it.

**Comment by ianka** [comment]
Please try [this latest commit.](https://github.com/ianka/xfs_undelete/commit/af2513e3d8d872113663eee200b9715e06982e0b)

**Comment by ianka** [comment]
Should be fixed with [v13.2](https://github.com/ianka/xfs_undelete/releases/tag/v13.2).

Feel free to reopen if the bug persists for you.
